src/Areas/Sql/Commands/AdAdmin/AdAdminListCommand.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Areas.Sql.Models;
+using AzureMcp.Areas.Sql.Options.AdAdmin;
+using AzureMcp.Areas.Sql.Services;
+using AzureMcp.Services.Telemetry;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.Sql.Commands.AdAdmin;
+
+public sealed class AdAdminListCommand(ILogger<AdAdminListCommand> logger)
+    : BaseSqlCommand<AdAdminListOptions>(logger)
+{
+    private const string CommandTitle = "List SQL Server AD Administrators";
+
+    public override string Name => "list";
+
+    public override string Description =>
+        """
+        Gets a list of Azure Active Directory administrators for a SQL server. This command retrieves all 
```

Comment:
Isn't Azure Active Directory renamed to Entra ID?

Reply:
yes

Reply:
@xiangyan99 and @wbreza yes it is however `az cli` still refers to it as `ad-admin` and thats where we went that route, but I hear you out. The portal calls it Entra admin so I think its fair to call it entra.

@jongio how difficult will it be to change it to `azmcp  sql server` **entra** `-admin list`

let me know, I will update the spec as well.

Reply:
We can't have dashes in the name, so it would be entraadmin list.